pkg/pillar/cmd/nim/nim.go

```diff
@@ -182,40 +152,18 @@ func Run() {
 	nimCtx.subGlobalConfig = subGlobalConfig
 	subGlobalConfig.Activate()
 
-	nimCtx.ManufacturerModel = model
-	nimCtx.DeviceNetworkConfig = &types.DeviceNetworkConfig{}
 	nimCtx.DevicePortConfig = &types.DevicePortConfig{}
-	item, _ := pubDevicePortConfigList.Get("global")
-	if item != nil {
-		dpcl := cast.CastDevicePortConfigList(item)
-		nimCtx.DevicePortConfigList = &dpcl
-		log.Infof("Initial DPCL %+v\n", nimCtx.DevicePortConfigList)
-	} else {
-		nimCtx.DevicePortConfigList = &types.DevicePortConfigList{}
-	}
-	nimCtx.DevicePortConfigList.CurrentIndex = -1 // No known working one
 	nimCtx.DeviceNetworkStatus = &types.DeviceNetworkStatus{}
 	nimCtx.PubDevicePortConfig = pubDevicePortConfig
 	nimCtx.PubDevicePortConfigList = pubDevicePortConfigList
 	nimCtx.PubDeviceNetworkStatus = pubDeviceNetworkStatus
-
-	// Get the initial DeviceNetworkConfig
-	// Subscribe from "" means /var/tmp/zededa/
-	subDeviceNetworkConfig, err := pubsub.Subscribe("",
-		types.DeviceNetworkConfig{}, false,
-		&nimCtx.DeviceNetworkContext)
-	if err != nil {
-		log.Fatal(err)
-	}
-	subDeviceNetworkConfig.ModifyHandler = devicenetwork.HandleDNCModify
-	subDeviceNetworkConfig.DeleteHandler = devicenetwork.HandleDNCDelete
-	nimCtx.SubDeviceNetworkConfig = subDeviceNetworkConfig
-	subDeviceNetworkConfig.Activate()
+	dnc := &nimCtx.DeviceNetworkContext
+	devicenetwork.IngestPortConfigList(dnc)
 
 	// We get DevicePortConfig from three sources in this priority:
 	// 1. zedagent publishing NetworkPortConfig
 	// 2. override file in /var/tmp/zededa/NetworkPortConfig/*.json
```

Comment:
Looks like we removed code that handles the override.json case. Should we remote this comment also?

Reply:
Same as above; has been in /config/DevicePortConfig and /var/tmp/zededa/DevicePortConfig.

I see that some comments refer to NetworkPortConfig, and those should be changed to refer to DevicePortConfig.